mgmtfn/k8splugin/contivk8s/k8s_cni_test.go

```diff
@@ -162,6 +164,17 @@ func setupTestServer() {
 		logger.Infof("k8s test plugin closing %s", driverPath)
 	}()
 
+	// make sure the listener is ready before returning
+	for count := 0; count < 5; count++ {
```

Comment:
Other way of fixing this might be to do ListenUnix inline in setupTestServer() and http.Serve() in a go routine.
That guarantees socket i.e. ready before you execute the tests